ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -154,6 +154,12 @@
 {%     endif %}
 
 ### Router BGP EVPN Address Family
+{%     if evpn_hostflap_detection.window is arista.avd.defined and evpn_hostflap_detection.threshold is arista.avd.defined %}
+#### EVPN Host Flapping Settings
```

Comment:
Please add an empty line between line 157 and 158. To give a consistent MD output.